src/pal/src/init/pal.cpp

```diff
@@ -1150,269 +1146,66 @@ Notes 2:
     This doesn't handle the case of directories with the desired name
     (and directories are usually executable...)
 --*/
-static LPWSTR INIT_FindEXEPath(LPCSTR exe_name)
+static LPWSTR INIT_EXEPath(LPCSTR exe_path)
```

Comment:
How about INIT_ConvertEXEPath? As I understand, the current implementation converts LPCSTR into LPWSTR.